### PR TITLE
H w7 cucumber

### PR DESCRIPTION
**Pull request by olkanevska**
Added and changed some methods in our module.rb.
Changed some methods in test_positive.rb.
Add solution in cucumber.

### CODE REVIEW

File: cucumber_project/features/add_user_in_project.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Feature: Add user in project
+
+  As a User
+  I want to add another user to my project
+
+  @positive
+  Scenario: Positive Add user in project
+    Given I am on Registration page
+    And I submit registration form with valid data
+    And I perform the actions necessary to create project
```
Comment (d-konovalov) [inline cucumber_project/features/add_user_in_project.feature]: ```I create project``` is better

File: cucumber_project/features/add_user_in_project.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    When I add to my project another use
```
Comment (d-konovalov) [inline cucumber_project/features/add_user_in_project.feature]: user?

File: cucumber_project/features/create_issue.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Feature: Create bug issue.
+
+  As a User
+  I want do random actions bug issue.
+  @positive
+  Scenario: Positive Create bug issue and add watcher
+    Given I am on Registration page
+    And I submit registration form with valid data
+    And I perform the actions necessary to create project
+    When I perform the actions necessary to random create or not bug issue
```
Comment (d-konovalov) [inline cucumber_project/features/create_issue.feature]: remove ```perform the actions necessary ...``` please

File: cucumber_project/features/create_project.feature [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+  As a User
+  I want to create project on the website as a registered user
+
+  @positive
+  Scenario: Positive Create project
+    Given I am on Registration page
+    And I submit registration form with valid data
+    When I perform the actions necessary to create project
+    Then I see a message that my actions led to the expected result
```
Comment (d-konovalov) [inline cucumber_project/features/create_project.feature]: ```I see expected message```

**Review by d-konovalov — COMMENTED** [review]
minor changes needed

File: cucumber_project/features/add_user_in_project.feature (hunk 1 above)
Comment (olkanevska) [reply cucumber_project/features/add_user_in_project.feature]: Сorrected

File: cucumber_project/features/add_user_in_project.feature (hunk 2 above)
Comment (olkanevska) [reply cucumber_project/features/add_user_in_project.feature]: Сorrected

File: cucumber_project/features/create_issue.feature (hunk 3 above)
Comment (olkanevska) [reply cucumber_project/features/create_issue.feature]: Corrected

File: cucumber_project/features/create_project.feature (hunk 4 above)
Comment (olkanevska) [reply cucumber_project/features/create_project.feature]: Сorrected

**Pull request merged**